prac_05/count_words.py

```diff
@@ -0,0 +1,15 @@
+""""Test String
+this is a test string to test the number of words counted is the to a a
+{num: word for num, word in enumerate(input_text)}
+"""
+
+input_text = input("String:")
```

Comment:
Delete from this point on, nah just kidding. very good.

Reply:
Ty